.github/workflows/forest.yml

```diff
@@ -0,0 +1,193 @@
+name: Integration tests
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - '*.md'
+      - '*.rb'
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '*.md'
+      - '*.rb'
+  schedule:
+    - cron: "0 0 * * *"
+
+env:
+  CI: 1
+  CARGO_INCREMENTAL: 1
+  CACHE_TIMEOUT_MINUTES: 5
+  SCCACHE_GHA_ENABLED: "true"
+  RUSTC_WRAPPER: "sccache"
+
+jobs:
+  build-macos:
+    name: Build MacOS
+    runs-on: macos-latest
+    steps:
+      - name: Checkout Sources
+        uses: actions/checkout@v3
+      - name: Setup sccache
+        uses: mozilla-actions/sccache-action@v0.0.3
+        timeout-minutes: ${{ fromJSON(env.CACHE_TIMEOUT_MINUTES) }}
+        continue-on-error: true
+      - name: Cargo Check
+        run: cargo check
+        env:
+          CC: "sccache clang"
+          CXX: "sccache clang++"
```

Comment:
Should this be moved to the global `env`?

Reply:
I guess we can try. Hopefully, it won't open any issues, those two variables tend to be *sensitive*.